workflows/eager.nf

```diff
@@ -215,6 +217,33 @@ workflow EAGER {
         ch_dedupped_flagstat = Channel.empty()
     }
 
+    //
+    // Subworkflow: Metagenomics screening
+    //
+
+    if( params.run_metagenomicscreening ) {
+        ch_bamfiltered_for_metagenomics.map{ meta, fastq ->
+            [meta+['single_end':true], fastq]
+        }
+        .set{ch_bamfiltered_for_metagenomics}
```

Comment:
Convention in eager is not to use `set` as it makes it harder to traceback channel usage, so 
```suggestion
        ch_bamfiltered_for_metagenomics = ch_bamfiltered_for_metagenomics
            .map{ meta, fastq ->
                [meta+['single_end':true], fastq]
            }
```